bark_ml/library_wrappers/lib_fqf_iqn_qrdqn/agent/base_agent.py

```diff
@@ -0,0 +1,267 @@
+from abc import ABC, abstractmethod
+import os
+import numpy as np
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+from bark_ml.library_wrappers.lib_fqf_iqn_qrdqn.memory import LazyMultiStepMemory, \
+    LazyPrioritizedMultiStepMemory
+from bark_ml.library_wrappers.lib_fqf_iqn_qrdqn.utils import RunningMeanStats, LinearAnneaer
+
+
+class BaseAgent(ABC):
+
+    def __init__(self, env, test_env, params):
```

Comment:
Please adhere to the coding guidelines: https://github.com/bark-simulator/bark-ml/blob/master/STYLE_GUIDE.md